### PR TITLE
Feat/start session

### PR DESCRIPTION
**Pull request by angelstackit**

### CODE REVIEW

File: src/components/InterviewDetail/InterviewDrawer.tsx [hunk 1]
```diff
@@ -16,9 +18,12 @@ import {
 import { FaUserAlt, FaUserPlus } from 'react-icons/fa'
 import { MdGroup } from 'react-icons/md'
 import Drawer from 'react-modern-drawer'
-import { Button } from '../common/Button'
 import { CardProfile } from '../common/CardProfile'
 import { SpinnerContainer } from '../common/SpinnerContainer'
+import { findInterviewById } from '@/features/app/actions/interview/findInterviewById/findInterviewById'
+import { openToastInfo } from '@/store/ui/ui.slice'
+// import { updateCandidate } from '@/features/app/actions/candidate/updateCandidate/updateCandidate'
```
Comment (juliswer) [inline src/components/InterviewDetail/InterviewDrawer.tsx:25]: remove commented code

File: src/components/UpcomingInterviews/UpcomingInterviewItem.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Image from 'next/image'
 import 'react-modern-drawer/dist/index.css'
 
 import { type Interview } from '@/common/types/Interview'
-import { getAvatarName } from '@/common/utils/get-avatar-name'
+// import { getAvatarName } from '@/common/utils/get-avatar-name'
```
Comment (juliswer) [inline src/components/UpcomingInterviews/UpcomingInterviewItem.tsx:5]: remove commented code

File: src/components/UpcomingInterviews/UpcomingInterviewItem.tsx [hunk 3]
```diff
@@ -52,10 +52,10 @@ export function UpcomingInterviewItem({ interview }: Props) {
     handleDrawerChange()
   }
 
-  const avatarName = getAvatarName(
-    interview.candidate?.firstName,
-    interview.candidate?.lastName
-  )
+  // const avatarName = getAvatarName(
```
Comment (juliswer) [inline src/components/UpcomingInterviews/UpcomingInterviewItem.tsx:55]: remove commented code

File: src/components/UpcomingInterviews/UpcomingInterviewItem.tsx [hunk 4]
```diff
@@ -96,7 +96,7 @@ export function UpcomingInterviewItem({ interview }: Props) {
         ) : (
           <div className="avatar placeholder h-full">
             <div className="w-8 rounded-full bg-neutral-focus text-neutral-content">
-              <span>{avatarName}</span>
+              {/* <span>{avatarName}</span> */}
```
Comment (juliswer) [inline src/components/UpcomingInterviews/UpcomingInterviewItem.tsx:99]: remove commented code

File: src/components/common/SpinnerContainer.tsx [hunk 5]
```diff
@@ -13,10 +13,11 @@ export function SpinnerContainer({ children, isLoading, style }: Props) {
       <div
         style={{
```
Comment (juliswer) [inline src/components/common/SpinnerContainer.tsx:14]: please avoid using the inLine styling. Use a css className instead or tailwind inline styling

File: src/pages/interview-session.tsx [hunk 6]
```diff
@@ -0,0 +1,143 @@
+// import { EmojisContainer } from '@/layout/EmojisContainer'
```
Comment (juliswer) [inline src/pages/interview-session.tsx:1]: delete commented code (check the entire file)

File: src/pages/sessions/[interviewId].tsx [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+              // Next Link href just accepts strings?
+              href={`${interview.candidate?.linkedin}`}
+            >
+              <FaLinkedin />
+              LinkedIn
+            </Link>
+            <Link
+              className="flex flex-row items-center"
+              // Todo
+              // Next Link href just accepts strings?
```
Comment (juliswer) [inline src/pages/sessions/[interviewId].tsx:104]: I dont really undestand the todo.
Next Link href only accepts strings

**Review by juliswer — CHANGES_REQUESTED** [review]
Basically. Remove commented code.
On the other hand, amazing job man! Let's keep it up

**Review by AlefrankM — APPROVED** [review]

**Pull request merged**